src/types/enums.rs

```diff
@@ -103,13 +103,17 @@ impl<'ctx> AnyTypeEnum<'ctx> {
             LLVMTypeKind::LLVMX86_FP80TypeKind |
             LLVMTypeKind::LLVMFP128TypeKind |
             LLVMTypeKind::LLVMPPC_FP128TypeKind => AnyTypeEnum::FloatType(FloatType::new(type_)),
+            #[cfg(feature = "llvm11-0")]
+            LLVMTypeKind::LLVMBFloatTypeKind => AnyTypeEnum::FloatType(FloatType::new(type_)),
```

Comment:
Any idea what this new type is for?

Reply:
BFloat : https://releases.llvm.org/11.0.0/docs/ReleaseNotes.html#changes-to-the-aarch64-backend
Looks like an Aarch64 (64 bit ARM) specific field

The scalable vector looks like a new generic IR type: https://releases.llvm.org/11.0.0/docs/ReleaseNotes.html#changes-to-the-llvm-ir